src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -143,9 +138,20 @@
                 name: 'language'
               }
             } only %}
-            <span class="ecl-site-header__language-code">{{ _language.code }}</span>
+            <span class="ecl-site-header__language-code">{{ _language_selector.code }}</span>
           </span>
         </a>
+        {% include '../ec-component-language-list/language-list-overlay.html.twig' with { 
+          items: _language_selector.overlay.items,
+          close_label: _language_selector.overlay.close_label,
+          title: _language_selector.overlay.title,
+          icon_path: _icon_file_path,
+          extra_attributes: [
+            { name: 'aria-labelledby', value: 'ecl-language-list__title' }, 
+            { name: 'role', value: 'dialog' },
+            { name: 'hidden', value: '' }
```

Comment:
I think we should improve the logic we have to set the extra_attributes, in ECL i see everytime these attributes that are not supposed to have any value but just to be set with an empty value "", we should be able to just set an attribute name (so we should check for the existence of the value key and only in that case print its value)

Reply:
Well, in this case we don't follow a suboptimal implementation in the ECL (remember that in the end that is just a demo...) Not asking you to fix it right now, (it's not enough not to pass the value), let's create a ticket for an improvement ... ;)